Subject: Upgrade gradle to 8.8
Author: Simon Frankenberger <simon-alpine@fraho.eu>
Upstream: https://github.com/plantuml/plantuml/issues/1543

This patch upgrades the gradle version to 8.8, fixing a SIGSEGV
on aarch64.

diff -ru old/gradle/wrapper/gradle-wrapper.properties new/gradle/wrapper/gradle-wrapper.properties
--- old/gradle/wrapper/gradle-wrapper.properties	2024-05-26 11:33:04.000000000 +0200
+++ new/gradle/wrapper/gradle-wrapper.properties	2024-07-04 07:58:33.915855795 +0200
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.8-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
